integration/src/test/resources/org/mule/test/construct/flow-ref.xml

```diff
@@ -251,5 +251,20 @@
     <sub-flow name="dynamicFlowRefWithSdkOperationInner">
         <http:request config-ref="requestConfig" path="/echo"/>
     </sub-flow>
-    
+
+    <flow name="foreachWithFlowRefAndMaxConcurrency">
+        <set-payload value="#[['apple', 'banana', 'orange']]"/>
```

Comment:
you may use `withPayload` in the flowRunner or the `collection` parameter in the `foreach`.